builds/misc/metrics-collector-images-publish.yaml

```diff
@@ -0,0 +1,60 @@
+pr: none
+
+variables:
+  NugetSecurityAnalysisWarningLevel: warn
+  
+jobs:
+  - deployment: publishImages
+    displayName: Publish Images
+    pool:
+      vmImage: ubuntu-18.04
+    workspace:
+      clean: all
+    environment: 'Azure-IoT-Edge-Core Release Env'
+    strategy:
+      runOnce:
+        deploy:
+          steps:
+          - checkout: self
+            clean: true
+            fetchDepth: 100
+        
+          - task: Bash@3
+            displayName: Log into Registries
+            inputs:
+              targetType: Inline
+              script: |
+                docker login $(registry.address) --username $(registry.user) --password $(registry.password)
+        
+          - task: Bash@3
+            displayName: 'Publish Metrics Collector - Linux amd64'
+            inputs:
+              filePath: '$(System.DefaultWorkingDirectory)/scripts/linux/moveImage.sh'
+              arguments: '--from $(registry.address)/$(from.registry.namespace)/azureiotedge-metrics-collector:$(version)-linux-amd64 --to $(registry.address)/$(to.registry.namespace)/azureiotedge-metrics-collector:$(version)-linux-amd64'
+      
+          - task: Bash@3
+            displayName: 'Publish Metrics Collector - Linux arm32v7'
+            inputs:
+              filePath: '$(System.DefaultWorkingDirectory)/scripts/linux/moveImage.sh'
+              arguments: '--from $(registry.address)/$(from.registry.namespace)/azureiotedge-metrics-collector:$(version)-linux-arm32v7 --to $(registry.address)/$(to.registry.namespace)/azureiotedge-metrics-collector:$(version)-linux-arm32v7'
+      
+          - task: Bash@3
+            displayName: 'Publish Metrics Collector - Linux arm64v8'
+            inputs:
+              filePath: '$(System.DefaultWorkingDirectory)/scripts/linux/moveImage.sh'
+              arguments: '--from $(registry.address)/$(from.registry.namespace)/azureiotedge-metrics-collector:$(version)-linux-arm64v8 --to $(registry.address)/$(to.registry.namespace)/azureiotedge-metrics-collector:$(version)-linux-arm64v8'
+
+          - task: Bash@3
+            displayName: 'Publish Metrics Collector - Windows amd64'
+            inputs:
+              filePath: '$(System.DefaultWorkingDirectory)/scripts/linux/moveImage.sh'
+              arguments: '--from $(registry.address)/$(from.registry.namespace)/azureiotedge-metrics-collector:$(version)-windows-amd64 --to $(registry.address)/$(to.registry.namespace)/azureiotedge-metrics-collector:$(version)-windows-amd64'
+      
+          - task: Bash@3
+            displayName: 'Publish Metrics Collector Manifest'
+            inputs:
+              targetType: filePath
+              filePath: '$(System.DefaultWorkingDirectory)/scripts/linux/buildManifest.sh'
+              arguments: '-r $(registry.address) -u $(registry.user) -p $(registry.password) -v $(version) -t $(System.DefaultWorkingDirectory)/edge-modules/azure-monitor/docker/manifest.yaml.template -n $(to.registry.namespace) --tags "$(tags)"'
```

Comment:
I have a high level question about how this works. I know that we publish images to mcr. But does mcr just mirror the `edgerelease/microsoft` location in our edgerelease acr?

This approach was taken from the way we publish the proxy.

Reply:
> does mcr just mirror...

Yes something like that